build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2141,4 +2141,7 @@ message RequestMetadata {
   // There is no expectation that this value will have any particular structure,
   // or equality across invocations, though some client tools may offer these guarantees.
   string configuration_id = 7;
+
+  // The details of other tools involved in handling the request, eg. client-side proxies.
```

Comment:
Please clarify whether tool_details is the handler closest to the user or closest to the server. For instance, if the request originates with Bazel and passes through proxy1 and proxy2 before reaching the server, should tool_details be Bazel or proxy2?

My opinion is that it should be Bazel as the originator of the request.